index.rst

```diff
@@ -1,83 +1,144 @@
+#. Related to the above two points, since the SEDs of many stars peak in g-band, and because the bandwidth is wider, most stars are brightest in g-band. With brighter stars, we should expect to see more dipoles above the noise floor.
 
-.. figure:: /_static/Dipoles_ds9_fourpanel_f012.png
-   :name: fig-dipole_fourpanel
-   :target: https://github.com/lsst-dm/dmtn-019/tree/master/_static/Dipoles_ds9_fourpanel_f012.png
+Defining a Dipole Severity Metric
+=================================
+
+In order to leverage the full utility of LSST software, I wanted to use only the source lists generated by source detection and measurement in processEimage.py and the DIA sources generated with imageDifference.py. A metric of dipole severity could be defined a great many different ways from the parameters in the source tables, but after extensive trials and testing the following, though far from perfect, appears to be robust:
+
+* First, define a reference flux to use as a normalization for the full catalog of sources extracted in standard source detection and measurement.
+
+* Next, calculate the total significance of both positive and negative sources in the difference image, and divide by the uncertainty in their flux.
+
+* Finally, take the logarithm of 1 plus the difference image source significance divided by the square root of the reference flux.
+  Dividing by the square root of the reference flux makes the quantity unitless, but the remainder of this step is fairly arbitrary and simply makes the output more easily human readable.
+
+
+In code, the above looks like::
+
+  ref_src = butler_ref.get("src", dataId=_id)      # load the source list with a butler
+  flux_ref = ref_src[fluxrefKey]                   # Extract a flux measurement, e.g. "base_PsfFlux_flux"
+  flux_ref = flux_ref[~ref_src[flagrefKey]]        # Remove flagged sources 
+  flux_ref = numpy.clip(flux_ref, 1.0, saturation) # Clip flux at saturation threshold (65000 counts)
+  flux_ref = np.sum(flux_ref)                   
+
+  dia_src = butler_dia.get("deepDiff_diaSrc", dataId=_id)                                  # load the difference image source list with a butler
+  combined_flux = (numpy.abs(dia_src[posFluxKey]) + numpy.abs(dia_src[negFluxKey]))        # Add the absolute value of positive and negative
+  significance = numpy.clip(combined_flux, 1.0, saturation) / numpy.abs(dia_src[sigmaKey]) # Clip at saturation threshold, and divide by the error in the flux measurement
+  significance = significance[~dia_src[flagKey]]                                           # Remove flagged sources
+  metric = numpy.log(1.0 + numpy.sum(metric) / numpy.sqrt(flux_ref))
+
+Finally, calculating the above dipole severity metric for the "Faint" and "Bright" simulations gives the following:
+
+Dipole Severity Metric Plots
+----------------------------
+
+.. figure:: /_static/Dipole_severity_metric_plot_test6.png
```

Comment:
These plots are pretty noisy. Would it be possible to run an ensemble of simulations and average the results to get a better feel for the trend and differences between the curves?